config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -318,6 +318,7 @@ presubmits:
     always_run: false
     optional: true
     decorate: true
+    run_if_changed: 'test\/e2e|templates\/test|scripts\/ci-conformance.sh|Makefile'
```

Comment:
Is this change and the one below it an omission from https://github.com/kubernetes/test-infra/pull/30933?

Reply:
it's intentional now that the PR has merged (#30933 was used to test the PR itself, we didn't want it to get triggered on any other PRs yet)